sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -812,6 +863,43 @@ public async Task StartRecognizeCustomFormsWithLabels(bool useStream, bool inclu
             Assert.AreEqual("948284", form.Fields[name].ValueData.Text);
         }
 
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        [Ignore("Need to get an approved file to test")]
```

Comment:
Are we following up on this?
Is this test still ignored?

Reply:
Yes. I keep pinging service team about it.
And yes, test is ignored